scripts/upload_test_results/requirements.txt

```diff
@@ -0,0 +1,6 @@
+certifi==2020.6.20
+configparser==5.0.0
+gitdb==4.0.5
+GitPython==3.1.7
+smmap==3.0.4
```

Comment:
what is that for?

Reply:
All that packages are required to run the script. I tested it using virtual env.